### PR TITLE
add recording rules to get inodes data

### PR DESCRIPTION
**Pull request by benjaminhuo**
add prometheus rules to get inodes data
verified in production env

### CODE REVIEW

File: rules/rules.libsonnet [hunk 1]
```diff
@@ -358,6 +358,34 @@
...
+          {
+            record: 'node:disk_inodes_total:',
+            expr: |||
+              max(
+                max(
+                  kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
+                ) by (node, host_ip)
+                * on (host_ip) group_right (node)
+                label_replace(
+                  (max(node_filesystem_files{mountpoint="/"}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
```
Comment (metalmatze) [inline rules/rules.libsonnet]: This label would be `mountpoint="/host/root"` for me. I guess we would need to make it configurable. Additionally we should also put the `nodeExporterSelector` to these matching labels.

File: rules/rules.libsonnet [hunk 2]
```diff
@@ -358,6 +358,34 @@
...
+              record: 'node:disk_inodes_free:',
+              expr: |||
+                max(
+                  max(
+                    kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
+                  ) by (node, host_ip)
+                  * on (host_ip) group_right (node)
+                  label_replace(
+                    (max(node_filesystem_files_free{mountpoint="/"}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
+                  )
```
Comment (metalmatze) [inline rules/rules.libsonnet]: Same as above.

File: rules/rules.libsonnet (hunk 1 above)
Comment (benjaminhuo) [reply rules/rules.libsonnet]: Got it, I'll figure out how to make it configurable

Comment (benjaminhuo) [reply rules/rules.libsonnet]: Yelp, these rules could be used for a dashboard.
Will make the changes as well. 

Thanks for the suggestion,
Ben

File: rules/rules.libsonnet (hunk 2 above)
Comment (benjaminhuo) [reply rules/rules.libsonnet]: I've push a new commit for this PR to address the concerns you raised.
Let me know if it's ok or not

Thanks
Ben

Comment (benjaminhuo) [reply rules/rules.libsonnet]: Is there anything else I should change ?

Comment (metalmatze) [reply rules/rules.libsonnet]: No, I just need some time to test the dashboard. :wink: :relaxed: Sorry.

**Comment by metalmatze** [comment]
screenshot from 2018-12-05 18-52-41

LGTM. Any more comments @brancz @tomwilkie?

**Comment by brancz** [comment]
lgtm

**Pull request merged**